requirements3-oidc.txt

```diff
@@ -0,0 +1,4 @@
+# A wrapper of Flask with pre-configured OIDC support.
+# License: Apache License 2.0
+# Upstream url: https://github.com/verdan/flaskoidc
+flaskoidc==0.0.2
```

Comment:
not sure if we need another requirement file or just put it under extra_require in the setup.py file(e.g https://github.com/lyft/amundsendatabuilder/pull/180).

Reply:
I like that idea a lot. Just pushed the commit. Could you re-approve it?

Thanks for the example!